client/src/components/CheckboxSelector.js

```diff
@@ -0,0 +1,43 @@
+import './CheckboxSelector.scss';
+
+export const FancyCheckboxSelector = ({items, legend, callbackToggle}) =>
+  <div className="legend-container">
+    <fieldset>
+      <legend className="checkbox-selector__legend">{legend}</legend>
```

Comment:
BEM nitpick, `checkbox-selector__legend` ends up as a sibling of a bunch of `checkbox-selector`s, not a child. Maybe make the containing div or fieldset `checkbox-selector`, and the current `checkbox-selector`s `checkbox-selector__option`s?